docs/tutorial/02-hello-world.mdx

```diff
@@ -70,8 +87,10 @@ Identifiers are custom and can be any name you want them to be to indicate what
 
 <Img src="https://storage.googleapis.com/flow-resources/documentation-assets/cadence-tuts/accounts-diagram.png" />
 
-A [Transaction](/cadence/language/transactions) in Flow is defined as an arbitrary-sized block of Cadence code that is signed by one or more accounts.
-Transactions have access to the `/storage/` and `/private/` domains of the accounts that signed the transaction and can read and write to those domains,
+A [Transaction](/cadence/language/transactions) in Flow is defined as an arbitrary-sized block of Cadence code
+that is signed by one or more accounts.
+Transactions have access to the `/storage/` and `/private/` domains of the accounts
+that signed the transaction and can read and write to those domains,
 as well as read and call functions in public contracts and the public domains in other users` accounts.
```

Comment:
> as well as read and call functions in public contracts and the public domains in other users` accounts.

Suggested: Transactions can also read and call functions in public contracts, and access public domains in other users' accounts. 

Reason: Run-on, clarity